configs/example_bmi_multi_realization_config_w_snow17.json

```diff
@@ -78,7 +78,7 @@
                                     "land_surface_radiation~incoming~shortwave__energy_flux": "DSWRF_surface",
                                     "land_surface_air__pressure": "PRES_surface",
 				    "ice_fraction_schaake" : "sloth_ice_fraction_schaake",
-				    "ice_fraction_xinan" : "sloth_ice_fraction_xinan",
+				    "ice_fraction_xinanjiang" : "sloth_ice_fraction_xinan",
```

Comment:
For consistency (with realization files in other repos) I would suggest to replace 
`"sloth_ice_fraction_xinan(1,double,1,node)": 0.0,` in the SLoTh block with
`"sloth_ice_fraction_xinanjiang(1,double,1,node)": 0.0,`

and then here use the mapping

`"ice_fraction_xinanjiang" : "sloth_ice_fraction_xinanjiang"`

Reply:
@aaraney Can you make these quick changes? I tried to quickly do it but GitHub told me no:

`You can’t commit to fix-cfe-name-mapping because it is a [protected branch](https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)`

Reply:
@SnowHydrology, yeah i'm not sure what is going on either. I used the `edit` feature on `github.com` to create this PR. Not sure if that is why we can't push to what is considered a `protected branch`? Let's just merge this and ill open another PR to address this.

Reply:
Just opened #41 to address this. That was really weird! Thanks for the suggestion @ajkhattak!